spec/index.bs

```diff
@@ -6320,11 +6371,20 @@ GPUQueue includes GPUObjectBase;
 
             **Returns:** {{undefined}}
 
-            If any of the following conditions are unsatisfied, generate a validation error and stop.
-            <div class=validusage>
-                - Every {{GPUBuffer}} referenced in any element of |commandBuffers| is in the
-                    `"unmapped"` [=buffer state=].
-                - Every [=usage scope=] contained in |commandBuffers| satisfies the [=usage scope validation=].
+            Issue the following steps on the [=Device timeline=] of |this|:
+            <div class=device-timeline>
+                1. If any of the following conditions are unsatisfied, generate a validation error and stop.
+                    <div class=validusage>
+                        - Every {{GPUBuffer}} referenced in any element of |commandBuffers| is in the
+                            `"unmapped"` [=buffer state=].
+                        - Every [=usage scope=] contained in |commandBuffers| satisfies the [=usage scope validation=].
```

Comment:
This part of the validation happens in `finish()`, there is no need to do it here

Reply:
That step was pre-existing, but happy to remove it here now that you've spotted it.

Reply:
Turns out this step didn't exist anywhere else in the doc, so I was getting an unexported reference error. Moved this step to `finish()`, because while it's stated that this should happen in `finish()` elsewhere in the prose it wasn't actually part of the validation steps on the method itself.